pxr/usd/usd/stageCache.cpp

```diff
@@ -143,15 +220,11 @@ struct DebugHelper
 
     bool IsEnabled() const { return _enabled; }
 
```

Comment:
Is the variadic template necessary here?
It looks like all callers of AddEntry pass the same UsdStageRefPtr, Id parameters.

Reply:
It's not necessary.